src/ThriveWorkerUnit.sol

```diff
@@ -66,6 +70,18 @@ contract ThriveWorkerUnit is ReentrancyGuard {
         _;
     }
 
+    /**
+     * @dev Modifier to check if the caller is the ThriveReviewFactory contract.
+     * This is - for now - only used to add a ThriveReviewContract as a validator.
+     */
+    modifier onlyThriveReviewFactory() {
```

Comment:
worker units can work without reviewer as well, reviewer is contract is actually validator